src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -123,6 +169,6 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * или сообщение "dateTime can't be formatted!"
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "Date can't be formatted!";
+        return dateTime.format(RANDOM_FORMATTER);
```

Comment:
you never return message "dateTime can't be formatted!"
complete this task according to description

Reply:
Done